lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -498,6 +528,7 @@ class JsonpMainTemplatePlugin {
 						"jsonpArray = jsonpArray.slice();",
 						"for(var i = 0; i < jsonpArray.length; i++) webpackJsonpCallback(jsonpArray[i]);",
 						"var parentJsonpFunction = oldJsonpFunction;",
+						`var trustedTypesPolicyName = "${trustedTypesPolicyName}"`,
```

Comment:
```suggestion
						`var trustedTypesPolicyName = ${JSON.stringify(trustedTypesPolicyName)}`,
```

Reply:
I see no reason why this can't be inlined into `getPolicy`

Reply:
You are right, it can be inlined.